test/steps/upload_request.go

```diff
@@ -0,0 +1,43 @@
+package steps
+
+import "encoding/json"
+
+type Output struct {
+	SourceSegments     bool `json:"source_segments,omitempty"`
+	TranscodedSegments bool `json:"transcoded_segments,omitempty"`
+}
+
+type OutputLocation struct {
+	Type    string `json:"type,omitempty"`
+	URL     string `json:"url,omitempty"`
+	Outputs Output `json:"outputs,omitempty"`
+}
+
+type UploadRequest struct {
```

Comment:
Wouldn't it be ok to just use the real struct defs: https://github.com/livepeer/catalyst-api/blob/9cb23e73717cd4f34c90d5a12606e7cb49d21348/handlers/upload.go#L38 ?

Reply:
It would, but I want to keep the test and real code separate - this will hopefully help catch weird cases around missing vs. blank parameters etc.